tests/retryable-reads/retryable-reads_error-001.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+Retryable reads: failure when disabled
```

Comment:
Similar to my comments on the other test, I think this can refer to the method being tested.